src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -1171,6 +1171,41 @@ Containers::Optional<MeshData> TinyGltfImporter::doMesh(const UnsignedInt id, Un
                 return Containers::NullOpt;
             }
 
+        /* Joint attribute ends with _0, _1 ... */
+        } else if(Utility::String::beginsWith(attribute.first, "JOINTS")) {
```

Comment:
TODO for testing: the code relies on the attributes being sorted (so JOINTS_0 before JOINTS_1) and without gaps. I think sorting is ensured implicitly due to how JSON dict data is stored, but the gaps need to be checked.